gen.go

```diff
@@ -193,9 +293,11 @@ func (f Field) Len() int {
 }
 
 type GenTypeInfo struct {
-	Name        string
-	Fields      []Field
-	Transparent bool
+	Name                string
+	SafeName            string   // Name of the type, but safe to use in a variable name
```

Comment:
InstantiatedTypeName?

Reply:
type isn't instantiated tho, going with "GeneratedName" for now unless you have a better suggestion

Reply:
Oh, I see.